CppExport/src/visitors/source_visitors/ExpressionVisitorSource.cpp

```diff
@@ -58,7 +61,7 @@ SourceFragment* ExpressionVisitorSource::visit(Expression* expression)
 			case PrimitiveTypeExpression::PrimitiveTypes::UNSIGNED_LONG: notAllowed(e); break;
 			case PrimitiveTypeExpression::PrimitiveTypes::FLOAT: *fragment << "float"; break;
 			case PrimitiveTypeExpression::PrimitiveTypes::DOUBLE: *fragment << "double"; break;
-			case PrimitiveTypeExpression::PrimitiveTypes::BOOLEAN: *fragment << "boolean"; break;
+			case PrimitiveTypeExpression::PrimitiveTypes::BOOLEAN: *fragment << "bool"; break;
```

Comment:
At some point we should really consider having a single expression visitor, since these things are unlikely to be different for the different cases.